server/controller/partyController.js

```diff
@@ -0,0 +1,35 @@
+import parties from '../db/dummy';
+
+import Helper from '../helper/util';
+
+class Party {
+  static createParty(req, res) {
+    const request = req.body;
+    if (!Helper.isParty(request)) {
+      res.status(400).json({
+        status: 400,
+        error: 'Invalid data post',
```

Comment:
What does invalid data post mean to an end user? When a `field` fails validation, you are expected to return a validation error for that particular field explicitly telling the user what is wrong with the field and not a generic message like this.